recipes/ipyopt/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "ipyopt" %}
+{% set version = "0.9.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 5a105ef68227304c0bb81c7583cf9ea83d27bfc3584a7a2e531fdef991d49e03
+
+build:
+  number: 0
+  skip: true  # [win]
+  script: "{{ PYTHON }} -m pip install --no-deps . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - pkg-config
+  host:
+    - python >=3
```

Comment:
Do you want to skip python 3? If you, I think it is better to skip it in the build section, as you are using the selectors to skip for windows